### DISCUSSION TITLE
希望有地方可以配置项目支持的文件后缀名

### DISCUSSION THREAD

**Discussion by constly** [ejoy/ant#98] [General]
我在接入 [cute](https://github.com/RandyGaul/cute_headers) 来播放声音，这个可以直接播放wav或者ogg文件。
问题在于 我没办法仿照加载bank文件的方式来加载wav，因为wav没在ant.vfs/std.lua的game_whitelist中。

我感觉我这种散人或者说小团队，一般不太倾向于去改引擎的代码，因为这意味着必须要维护一个引擎分支，很麻烦，
如果可以在项目目录配置 或者 有程序接口可以设置那就比较方便了。

**Comment by actboy168** [comment]
feature request请提issue。

**Comment by constly** [comment]
已解决, 新增 .vfsignore文件，里面加上:
whitelist:
	wav
